tests/forms_tests/tests/test_formsets.py

```diff
@@ -602,16 +630,19 @@ def test_formsets_with_ordering(self):
             ],
         )
 
-    def test_formsets_with_order_custom_widget(self):
-        class OrderingAttributFormSet(BaseFormSet):
+    def test_formsets_with_ordering_custom_widget(self):
+        """
+        Asserts that the custom ordering widget is correctly applied.
+        """
```

Comment:
To add more explanation, docstrings should state the expected behavior and omit prefixes like "Asserts that" since all tests assert things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style). TBH, I don't see much value in this docstring, please revert it.